### ISSUE TITLE
Pulseeffects-3.2.3 crashes at start when projectm is running

### ISSUE THREAD

**Issue by AndrewAmmerlaan** [wwmm/pulseeffects#215]
Hi, 

I noticed today that when projectm (media-sound/projectm-pulseaudio-2.1.0) is already running and I try to start pulseeffects it crashes with:
`ERROR:/var/tmp/portage/media-libs/gst-plugins-good-1.12.4/work/gst-plugins-good-1.12.4/gst/spectrum/gstspectrum.c:991:gst_spectrum_transform_ip: assertion failed: (size == 0)
Aborted
`
Otherwise it works great, I run KDE (kde-apps/kde-meta-18.04.0, kde-plasma/plasma-desktop-5.12.5) on a gentoo linux system (kernel version 4.16.7), and I should probably also add that I patched pulseaudio-11.1 to fix issue #99 

This seems to be a minor issue, I don't really use projectm often anyway. But since I couldn't find a bugreport on it I figured I'd just go ahead and add one.

**Comment by wwmm** [comment]
Does this program record audio? I think that a few audio recording apps are not playing nice with some changes I made to the way PulseEffects sets latency. What values to you have for latency in PE settings menu?

**Comment by AndrewAmmerlaan** [comment]
It doesn't exactly record audio, but it is a sound visualization thingy, so I it probably does somehow monitor the audio. Latency is set to 10000 in output effects and 15955 in input effects, I have downloaded some presets but I haven't changed these settings myself.

**Comment by wwmm** [comment]
Is this still a problem in the latest version?

**Comment by AndrewAmmerlaan** [comment]
Well projectm-pulseaudio got removed from the gentoo repos because it didn't get ported to qt5, and therefore it (and all other qt4 stuff) is no longer on my system. 
I have tested the vlc implementation of projectm which does not cause pulseeffects to crash. I suspect this is because the vlc implementation is not systemwide (it only does visualizations for whatever is playing in vlc) and projectm is therefore run before the audio goes to pusleeffect. 

If anyone else runs into issues like this with projectm they can use vlc instead of a systemwide implementation, and because the relevant application is no longer in active development this can probably be closed.

As a side note, when using vlc and pulseeffects together it is probably a good idea to disable any effects in the vlc settings menu to prevent double application of any effects.
